agent/spec/lib/kontena/workers/node_info_worker_spec.rb

```diff
@@ -214,4 +214,22 @@
       expect(subject.private_ip).to eq('192.168.2.10')
     end
   end
+
+  describe '#publish_node_stats' do
+    it 'sends stats via rpc' do
+      expect(rpc_client).to receive(:notification).once do |key, msg|
```

Comment:
Awesome, let me take a look.  I was having issues with hash_including but I probably was doing it wrong.